tm2/pkg/crypto/keys/client/version_test.go

```diff
@@ -0,0 +1,48 @@
+package client
+
+import (
+	"testing"
+	"context"
+
+	"github.com/gnolang/gno/tm2/pkg/commands"
+	"github.com/gnolang/gno/tm2/pkg/version"
+	"github.com/gnolang/gno/tm2/pkg/testutils"
+	"github.com/stretchr/testify/require"
+)
+
+func TestVersionApp(t *testing.T) {
+	t.Parallel()
+
+	// make new test dir
+	kbHome, kbCleanUp := testutils.NewTestCaseDir(t)
+	defer kbCleanUp()
```

Comment:
Do we need this? (is some leftover kb generated on disk if we don't explicitly specify it and clean it up?)